ui/apps/platform/cypress/integration/violations/violations.test.js

```diff
@@ -99,17 +55,29 @@ describe('Violations page', () => {
     });
 
     it('should have Entity column in table', () => {
```

Comment:
What do you think about combining this and the following test? It seems like they are both testing small variations on the same thing. Might save us ~1 second on test runs to not have to do an extra page navigation 😄

Reply:
Indeed, there is one test for all table column headings in other containers.

Only because the number of changes exceeded my original intention, I will save this for a follow up.